Add SteelSeries docs

th3an7 opened the pull request:
- Changes were tested locally - seems to load images fine (maybe they are too big?)
- Let me know if any edits are needed

th3an7 committed:
Assets for SteelSeries docs

th3an7 committed:
Added SteelSeries docs

th3an7 committed:
SteelSeries docs update

...so it works for website (i think)

diogotr7 commented:
looks good for a start. might be worth mentioning the weird mousepad stuff and the rival 600?

th3an7 committed:
Create Known Issues for SS...

...about mices with more than 2 zones (Rivals 600 and 650)

th3an7 committed:
Update Known Issues for SS

This should make it clearer for other devices too
- Rival 600/650 (side bars)
- Zoned Apex keyboards
- MSI Monitors
- MSI Z270 Gaming Pro Carbon motherboard
